code/WorkInProgress/recycling/conveyor.dm

```diff
@@ -764,6 +925,61 @@ ADMIN_INTERACT_PROCS(/obj/machinery/conveyor_switch, proc/trigger)
 				C.setdir()
 		SEND_SIGNAL(src,COMSIG_MECHCOMP_TRANSMIT_SIGNAL,"switchTriggered")
 
+/obj/machinery/conveyor_switch/built/
+	desc = "A conveyor control switch. This one looks like it was built recently."
+	var/static/list/switch_ids
```

Comment:
i don't see where this list is used really
does it only exist because if switches are built on the same tick the TIME would be the same?

Reply:
Yeah. On hindsight, that's almost impossible from happening. I'll delete the code, tell me if it should be put back again;

Reply:
Yeah. On hindsight, that's almost impossible from happening. I'll delete the code, tell me if it should be put back again.

Reply:
I mean, it could still happen. I don't see how it could happen though with this current implementation. It could be solved cleaner if it's an issue.

Reply:
Probably won't happen, and it is easily fixable if it does happen (two switches with the same ID can, and is supposed to happen in the PR).